learn/guides/ballerina-openapi-support.md

```diff
@@ -192,48 +217,64 @@ Client generated successfully. The following files were created.
 -- types.bal
 -- utils.bal
 ```
-### Generate a Ballerina client with boiler-plate tests
+### Generate with boiler-plate tests
+
 Use the `--with-tests` flag in the client mode to generate a Ballerina client with boilerplate test cases for all the remote functions available in it.
 ```bash
 $ bal openapi -i <openapi-contract> [--mode client] [--with-tests]
 ```
 
-**Example:**
+For example,
+
 ```bash
 $ bal openapi -i hello.yaml --mode client --with-tests
 ```
 In addition to the above-mentioned generated file, this will generate a `test.bal` file in the default client generation.
 
 ### Generate with nillable types
-This is an optional flag in the OpenAPI to Ballerina command. If your OpenAPI specification includes JSON schema properties that are not marked as **nullable:true**, they may be returned as null in some responses which will result in a JSON schema to Ballerina record data binding error. If you suspect this can happen for any property, it is safe to generate all data types in the generated record with Ballerina nil support by turning this flag on.
+
+>**Info:** This is an optional flag in the OpenAPI to Ballerina command.
+
+If your OpenAPI specification includes JSON schema properties that are not marked as **nullable:true**, they may be returned as null in some responses which will result in a JSON schema to Ballerina record data binding error. If you suspect this can happen for any property, it is safe to generate all data types in the generated record with Ballerina nil support by turning this flag on.
+
 ```bash
 $ bal openapi -i <openapi-contract> [-n |--nullable]
 ```
 
->**Info:** For more command options, see [OpenAPI to Ballerina command reference](#openapi-to-ballerina-command-reference).
+For more command options, see [Command reference](#command-reference).
```

Comment:
if we have a separate reference for `Ballerina to OpenAPI command`, we better not rename this with a generic name like this. @lnash94 WDYT?

Reply:
Yeah , we have separate command `Ballerina to OpenAPI command`, @praneesha is it possible to keep the existing title since this section contains `OpenAPI to Ballerina command reference` 🙂

Reply:
I think we should have 2 topics as `Ballerina to OpenAPI command` and `OpenAPI to Ballerina command` and move both reference topics to the References Learn section.